lib/tokenizers/native.ex

```diff
@@ -10,7 +10,8 @@ defmodule Tokenizers.Native do
     crate: "ex_tokenizers",
     version: version,
     base_url: "#{github_url}/releases/download/v#{version}",
-    force_build: System.get_env("TOKENIZERS_BUILD") in ["1", "true"]
+    # force_build: System.get_env("TOKENIZERS_BUILD") in ["1", "true"]
+    force_build: true
```

Comment:
```suggestion
    force_build: System.get_env("TOKENIZERS_BUILD") in ["1", "true"]
```

If you want to force build, add a `-dev` to the version in the mix.exs.

Reply:
Thank you!